### PR TITLE
Updated hacked FAQ page and dependencies

### PR DESCRIPTION
**Pull request by edkeziah**
This pull request implements a hacked version of the FAQ page. Please review the changes and let me know if there are any issues or concerns.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -40,3 +40,6 @@ utils/db/serviceAccountKey.json
 
 # ENV
 .env
+
+# yarn
+.yarn
```
Comment (sweetmantech) [inline .gitignore:45]: nice addition 🔥

File: .yarnrc.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```
Comment (sweetmantech) [inline .yarnrc.yml]: what's the purpose of this change?

Is this file needed? Or, was it an accidental addition?

File: pages/faq/index.tsx [hunk 3]
```diff
@@ -1,65 +1,77 @@
...
+/* eslint-disable @next/next/no-img-element */
 import Contact from "../../components/Contact"
 import Footer from "../../components/Footer"
 import Header from "../../components/Header"
-import customLoader from "../../lib/customLoader"
 
 const FAQ = () => (
   <div className="relative z-0 w-full h-screen">
     <Header contactHref="#contact" />
+    {/* Add the hacked images */}
```
Comment (sweetmantech) [inline pages/faq/index.tsx]: remove comments.

philosophy: code should be self-documenting

Comment (sweetmantech) [reply pages/faq/index.tsx]:
```suggestion
```

File: pages/faq/index.tsx [hunk 4]
```diff
@@ -1,65 +1,77 @@
...
+      <img
+        src="/hacked/hackedImage2.png"
+        className="w-[560px] md:w-[1050px] absolute top-[800px] md:top-[1450px]"
+        alt="Hacked 2"
+      />
+      <img 
+        src="/hacked/hackedImage3.png" 
+        className="absolute md:top-[300px] top-[1150px]"
+        alt="Hacked 3" />
+    </div>
```
Comment (sweetmantech) [inline pages/faq/index.tsx]: this is fine for now, however, I think best-practice here is to abstract this code to a new component.

I'd call it `<HackedFaqOverlay />`

philosophy: [A module should be responsible to one, and only one, actor.](https://en.wikipedia.org/wiki/Single-responsibility_principle)

File: package.json [hunk 5]
```diff
@@ -28,7 +28,11 @@
     "ethers": "^5.0.0",
     "express": "^4.18.2",
     "framer-motion": "^9.0.2",
+<<<<<<< HEAD
+    "magic-sdk": "^16.2.0",
+=======
     "loglevel": "^1.8.1",
+>>>>>>> a3ad70eadab0bf34424d4581ff83e50e161ebf4f
```
Comment (sweetmantech) [inline package.json]: you need to resolve the merge conflicts.

**Review by sweetmantech — CHANGES_REQUESTED** [review]
`package.json` needs to have merge conflicts resolved

File: .yarnrc.yml (hunk 2 above)
Comment (edkeziah) [reply .yarnrc.yml]: I added the `nodeLinker: node-modules` setting to the `.yarnrc.yml` file, but I don't remember the specific reason for doing so. I've reviewed the codebase history and documentation, but I couldn't find any information that would provide me with the necessary context.

Could you please help me understand why this setting was added? I want to make sure that my changes are in line with the team's standards and practices.

Thanks in advance for your help!

Comment (crazzywizard) [reply .yarnrc.yml]: This was done as an attempt to figure out why `yarn install` was creating .yarn folder and fix it.

**Review by sweetmantech — APPROVED** [review]

**Pull request merged**